src/PKSim.Core/Commands/NormalizeRelativeExpressionCommand.cs

```diff
@@ -13,53 +13,58 @@ namespace PKSim.Core.Commands
    ///    Liver : 10
    ///    Kidney :10
    ///    Stomach:20
-   ///    The normlized result would be
+   ///    The normalized result would be
    ///    Liver : 0.5
    ///    Kidney :0.5
    ///    Stomach:1
    /// </summary>
    public class NormalizeRelativeExpressionCommand : BuildingBlockChangeCommand
    {
       private IndividualMolecule _molecule;
+      private ISimulationSubject _simulationSubject;
       private readonly string _moleculeId;
 
-      public NormalizeRelativeExpressionCommand(IndividualMolecule molecule, IExecutionContext context)
+      public NormalizeRelativeExpressionCommand(IndividualMolecule molecule, ISimulationSubject simulationSubject, IExecutionContext context)
```

Comment:
This command is now only called when getting values from DB (in fact they are already normalized it seems but better safe than sorry)